Include/internal/pycore_pylifecycle.h

```diff
@@ -108,8 +108,7 @@ PyAPI_FUNC(PyObject*) _PyErr_WriteUnraisableDefaultHook(PyObject *unraisable);
 
 PyAPI_FUNC(void) _PyErr_Print(PyThreadState *tstate);
 PyAPI_FUNC(void) _PyErr_Display(PyObject *file, PyObject *exception,
-                                PyObject *value, PyObject *tb);
-
+                                PyObject *value, PyObject *tb);                           
```

Comment:
nitpick: it seems like you removed a newline or something else. you may revert that.